dao/mongodb.rya/src/main/java/org/apache/rya/mongodb/instance/MongoDetailsAdapter.java

```diff
@@ -124,11 +123,6 @@ private static DBObject toDBObject(final PCJIndexDetails pcjIndexDetails) {
         // Is Enabled
         builder.add(PCJ_ENABLED_KEY, pcjIndexDetails.isEnabled());
 
-        // Fluo Details if present.
-        if(pcjIndexDetails.getFluoDetails().isPresent()) {
```

Comment:
I don't think we should be deleting conversion code just because we do not support this piece of information in Mongo DB currently. The class advertises itself as a converter from RyaDetails object to BasicDBObject. Either the RyaDetails used by Mongo should be different from that used by Accumulo (I don't think this is the route we should go), or some other hunk of code should ensure the fluo details are never present before it even gets to this point.

Reply:
This seems like a bigger problem outside the scope of this.  I deleted it because tests were failing throughout installconfig expecting to see fluo stuff and it not being there.  Adding it back in would require changes to be made elsewhere to get glue code